File: Striver Series/5RemoveDuplicatesFromInPlaceOfSortedArray.cpp
#include<iostream>
#include<set>
using namespace std;
class Solution{
    public:
    int solv(int arr[],int size){
        set<int>store;
        for(int i=0;i<size;i++){
            store.insert(arr[i]);
        }
        for(auto i:store){
            cout<<i<<" ";
        }
    }
};
int main(){
int arr[]={1,1,2,2,2,3,3};
int size= sizeof(arr)/sizeof(arr[0]);
Solution sl;
sl.solv(arr,size);
}

File: python/5RemoveDuplicatesFromInPlaceOfSortedArray.py
class Solution:
    def solv(self,list):
        store=set()
        for i in list:
            store.add(i)
        result=list(store)
        print(' '.join(map(str, result)))
        print(store)
list=[1,1,2,2,2,3,3]
sl=Solution()
sl.solv(list)